src/utils/send-stream-experimental.js

```diff
@@ -0,0 +1,199 @@
+'use strict'
+const { Duplex, Transform } = require('stream')
+const isStream = require('is-stream')
+const nanoid = require('nanoid')
+const pump = require('pump')
+const Multipart = require('./multipart-experimental')
+const { prepareWithHeaders } = require('./prepare-file')
+
+/** @private @typedef {import("../files/add-experimental").AddOptions} AddOptions */
+
+const noop = () => {}
+
+/**
+ * Convert back to the proper schema
+ *
+ * @private
+ * @param {Object} data
+ * @returns {Object}
+ */
+const convert = (data) => {
```

Comment:
Since this is `private`, it should be named `_arrayToStream`. However, it can also be moved to a `util`. Same below with `prepareTransform`